src/Sulu/Bundle/CategoryBundle/Tests/Unit/DependencyInjection/DeprecationCompilerPassTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\CategoryBundle\Tests\Unit\DependencyInjection;
+
+use Matthias\SymfonyDependencyInjectionTest\PhpUnit\AbstractCompilerPassTestCase;
+use Sulu\Bundle\CategoryBundle\DependencyInjection\DeprecationCompilerPass;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+
+/**
+ * Test the deprecation transformation compiler pass.
+ */
+class DeprecationCompilerPassTest extends AbstractCompilerPassTestCase
```

Comment:
Stupid question: What do we need the `DeprecationCompilerPass` for?  We are using the persistence bundle here now, and the parameters being set there are already set everytime, aren't they?